index.js

```diff
@@ -0,0 +1,141 @@
+const inputTitle = document.querySelector('.title');
+const inputAuthor = document.querySelector('.author');
+const addBtn = document.querySelector('.add-btn');
+const sectionBooks = document.querySelector('.books');
+const footer = document.getElementById('footer');
+
+const findHight = () => {
+  if (window.innerHeight > document.body.scrollHeight) {
+    footer.style.position = 'fixed';
+  } else {
+    footer.style.position = 'relative';
+  }
+};
+
+function loadBooks(bookCollection) {
+  sectionBooks.innerHTML = '';
+  /* eslint-disable */
+  if (bookCollection.length > 0) {
+    bookCollection.forEach((book, i) => {
+      sectionBooks.insertAdjacentHTML(
+        'afterbegin',
+        books.returnBook(book.title, book.author, i)
+      );
+    });
+    console.log(window.innerHeight);
+    console.log(document.body.scrollHeight);
+  }
+  books.colorGiver();
+  /* eslint-enable */
+}
+
+class BookApp {
+  height = 0;
+
+  constructor() {
+    /* eslint-disable */
```

Comment:
- [ OPTIONAL] Kindly over  eslint disable line and instead get used to fix the issue in the code manually